test/services/keyVault/keyVault-storage-tests.js

```diff
@@ -0,0 +1,228 @@
+//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+
+const should = require('should');
+const util = require('util');
+const Testutil = require('../../util/util');
+const msRestAzure = require('ms-rest-azure');
+const AzureStorage = require('azure-storage');
+const SuiteBase = require('../../framework/suite-base');
+const KeyVaultManagementClient = require('../../../lib/services/keyVaultManagement/lib/keyVaultManagementClient');
+const AuthorizationManagementClient = require('../../../lib/services/authorizationManagement/lib/authorizationManagementClient');
+const StorageManagementClient = require('../../../lib/services/storageManagement2/lib/storageManagementClient');
+const testPrefix = 'keyvaultstorage-tests';
+const groupPrefix = 'nodeTestGroup';
+const vaultPrefix = 'testacc';
+const createdGroups = [];
+const createdVaults = [];
+const createdStorageAccounts = [];
+const KeyVault = Testutil.libRequire('services/keyVault');
+
+const requiredEnvironment = [
+  { name: 'AZURE_TEST_LOCATION', defaultValue: 'westus' },
+];
+
+var suite;
+var kvManagementClient;
+var storageMgmtClient;
+var groupName;
+var acclocation;
+var akvClient;
+var testVault;
+var userOid;
+var testStorageAccount;
+
+var _tokenFromUserCredentials = () => { 
+  return new Promise ( (resolve, reject) => { 
+    if(suite.isPlayback) {
+      resolve( { 'oid': '7541419d-883d-452f-a823-56aa8bf0749f' } );
```

Comment:
>resolve( { 'oid': '7541419d-883d-452f-a823-56aa8bf0749f' } ); [](start = 6, length = 61)

you should avoid hardcoding values here that might change when the tests are re-recorded.  possibly drop to a file when running live and read when playback?